packages/gtk_doc.rb

```diff
@@ -1,38 +1,51 @@
 require 'package'
 
+# There are alot of patches to grab here ~ They've all come from Void-Packages (xbps-src)
+
 class Gtk_doc < Package
-  description 'GTK-Doc is a project which was started to generate API documentation from comments added to C code.'
-  homepage 'https://www.gtk.org/gtk-doc'
-  version '1.29'
+  description 'Documentation tool for public library API'
+  homepage 'https://www.gtk.org/gtk-doc/'
+  version '1.32'
   compatibility 'all'
-  source_url 'https://github.com/GNOME/gtk-doc/archive/GTK_DOC_1_29.tar.gz'
-  source_sha256 'fdf5051e1f164fc1454a4530d217ee2c09dcc2c8e42b93cd5d68645493319ce5'
-
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk_doc-1.29-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gtk_doc-1.29-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gtk_doc-1.29-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gtk_doc-1.29-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '8641b7d74ba0de51e40347751c5022164f490703fcab9902d85843a3c6a8df41',
-     armv7l: '8641b7d74ba0de51e40347751c5022164f490703fcab9902d85843a3c6a8df41',
-       i686: '6c14eadd88bb454b2bdd1212935095d6fe478e932bc6bab301d519e767217c2f',
-     x86_64: '53adc04a62779005099e257fc2ba6df76c0b8a9dce8cc31bd6eac6f4181b69c4',
-  })
+  source_url 'https://ftp.gnome.org/pub/GNOME/sources/gtk-doc/1.32/gtk-doc-1.32.tar.xz'
+  source_sha256 'de0ef034fb17cb21ab0c635ec730d19746bce52984a6706e7bbec6fb5e0b907c'
 
-  depends_on 'six'
+  depends_on 'docbook_xml'
+  depends_on 'docbook_xsl'
+  depends_on 'itstool'
   depends_on 'libxslt'
+  depends_on 'docbook_xml43'
 
+  def self.patch
+    puts
+    puts 'Grabbing patches'.lightblue
+    system 'curl --ssl --insecure -L -o "output-reproducible.patch" "https://git.io/JUlWD" --progress-bar'
```

Comment:
Why do we need the `--insecure` option for https?

Reply:
We don't idk exactly why I have that there, I'll remove that.